services/github/github-constellation.js

```diff
@@ -5,6 +5,11 @@ import GithubApiProvider from './github-api-provider.js'
 import { setRoutes as setAdminRoutes } from './auth/admin.js'
 import { setRoutes as setAcceptorRoutes } from './auth/acceptor.js'
 
+const readPackagesScope = 'read:packages'
+// Multiple scopes need to be uri-encoded space delimited
+const tokenScopes = `${readPackagesScope}`
+const persistenceScopeDelimiter = '.scopes.'
```

Comment:
realize that defining these up here has some impact on readability in places where they are used, but i opted to encapsulate and hoist them up here after twice failing to type the trailing `.` when using the raw delimiter value in each site